lib/feature/browser/browser_tab_view/browser_tab_view.dart

```diff
@@ -0,0 +1,147 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_inappwebview/flutter_inappwebview.dart';
+
+class BrowserTabView extends StatefulWidget {
+  const BrowserTabView({required this.onScroll, super.key});
+
+  final void Function({
```

Comment:
Imho it's better to make function description via `typedef`

Reply:
Fixed, thnx!